src/graphics/embeds.ts

```diff
@@ -209,23 +209,44 @@ export function ehentaiBookPreviewEmbed(
     const embed = info(client, "「以下是這本魔法書的相關資訊...」");
     embed.setImage(galleryMetadata.thumb);
 
-    // resolve tags and translate
+    //merge the tag with same keys
+    const tagMap = new Map<string, string[]>();
+    galleryMetadata.tags.forEach((element) => {
+        const tag = element.split(":");
+        const tagList = tagMap.get(tag[0]);
+        if (tagList) {
+            tagMap.set(tag[0], [...tagList, tag[1]]);
```

Comment:
```suggestion
            tagList.append(tag[1]);
```

Using spread operator cause shallow copy.